qutebrowser/browser/hints.py

```diff
@@ -130,7 +139,21 @@ def _move_to_elem(self):
     def cleanup(self):
```

Comment:
Could you add:
```
        self.setParent(None)
```
somewhere in here? When the hint is returned to the pool, it currently still has the tab it was displayed in as a parent, so if that tab is deleted, it will delete this hint as well (leading to a crash later on when it's accessed).